datahub-web-react/src/app/shared/admin/HeaderLinks.tsx

```diff
@@ -107,6 +110,20 @@ export function HeaderLinks(props: Props) {
                     </Link>
                 </LinkWrapper>
             )}
+            {showManagePosts && (
+                <LinkWrapper>
+                    <Link to="/posts">
+                        <Button id="manage-posts-page" type="text">
+                            <Tooltip title="Manage home page posts">
+                                <NavTitleContainer>
+                                    <NotificationOutlined />
+                                    <NavTitleText>News</NavTitleText>
+                                </NavTitleContainer>
+                            </Tooltip>
+                        </Button>
+                    </Link>
+                </LinkWrapper>
```

Comment:
after talking with our designer we have some concerns with adding another button to the top header of the app. Instead, we suggest putting this in Settings under the left sidebar "Manage" section right underneath the new Ownership Types section. then we can render the homepage posts there in settings